packages/ERTP/src/amountMath.js

```diff
@@ -7,6 +7,19 @@ import mathHelpersLib from './mathHelpersLib';
 
 import './types';
 
+// We want an enum, but narrowed to the AmountMathKind type.
+/**
+ * Constants for the kinds of amountMath we support.
+ * @type {{ NAT: 'nat', SET: 'set', STRING_SET: 'strSet' }}
+ */
+const MathKind = {
```

Comment:
Hmm, yeah, I think with this, we can get rid of the importManager usage.

Reply:
done